.github/workflows/ci.yaml

```diff
@@ -1,6 +1,6 @@
 ---
 name: build
-on: push
+on: [push, pull_request]
```

Comment:
Another way to do this is to add branch restrictions for push. This avoids building for everything that gets pushed.

```
  push:
    branches:
      - master
    tags:
      - v*.*
  pull_request:
```

Reply:
I was just trying to get this to run for PRs on forks (like this one), not add restrictions.